.circleci/config.yml

```diff
@@ -184,9 +196,12 @@ jobs:
       - save-cache: *cache_save_gradle_build
 
     # Build and test
+      - run:
+          name: Bundle
+          command: mkdir example/android/app/src/main/assets && npx react-native bundle --platform android --dev false --entry-file example/index.js --bundle-output example/android/app/src/main/assets/index.android.bundle --assets-dest example/android/app/src/main/res/
```

Comment:
I wonder why you have to do this?

Reply:
Yeah, I thought the same, but hit this error when building release, https://github.com/facebook/react-native/issues/11696 . That's the only workaround I found, let me know if guys know better ways to resolve the issue...

Reply:
I guess it's fine as it's for example app anyway